### PR TITLE
Increase nose output verbosity with -v... argument

### PR DESCRIPTION
**Pull request by millerdev**
When you're running tests and you want more output.

@dimagi/team-commcare-hq

### CODE REVIEW

File: manage.py [hunk 1]
```diff
@@ -87,6 +87,40 @@ def set_default_settings_path(argv):
...
+        -vvv: do not capture logging
+        -vvvv: enable nose internal logging
+    """
+    import logging
+
+    def set_verbosity(arg, i):
+        args = []
+        verbosity = sum(1 for c in arg if c == "v") + 1
+        if len(arg) > verbosity:
+            # preserve other single-letter arguments (ex: -xv)
```
Comment (esoergel) [inline manage.py]: This seems like it shouldn't need to be done manually - is there any way we could use `argparse` or similar to handle this sort of thing?

**Review by esoergel — APPROVED** [review]
I exclusively do the equivalent of `-vvv`:
```
alias djtest='REUSE_DB=1 ./manage.py test --nocapture --nologcapture --noinput'
```
It's nice to keep output low when running a full test suite, but almost all of the times I run tests locally, I'm running only a handful, and nose's attempts to silence everything and playback some of it later are a huge hindrance to debugging.

File: manage.py (hunk 1 above)
Comment (millerdev) [reply manage.py]: argparse is used internally by django and nose. I'm not sure there's a good way to use it to parse, mutate, and re-constitute a set of arguments that would be easier than what this code is doing.

**Comment by snopoke** [comment]
This should be announced to the team since it changes workflow.

**Comment by millerdev** [comment]
@snopoke I did @ mention dimagi/team-commcare-hq. Were you thinking of some other announcement channel?

**Comment by snopoke** [comment]
ah, that seems fine. I must have missed the notification.

**Pull request merged**